meta-resin-common/recipes-connectivity/openvpn/openvpn/prepare-openvpn.service

```diff
@@ -9,4 +9,4 @@ RemainAfterExit=yes
 ExecStart=@BINDIR@/prepare-openvpn
 
 [Install]
-WantedBy=resin.target
```

Comment:
I don't really understand how this mechanism works. config-json.service gets activated only on path changed. So This service will never be activated unless the patch changed? It's a little confusing and the entire explanation on how services are triggered should reside in the commit log.